modules/services/pass-secret-service.nix

```diff
@@ -2,31 +2,47 @@
 
 with lib;
 
-let serviceCfg = config.services.pass-secret-service;
+let cfg = config.services.pass-secret-service;
 in {
   meta.maintainers = [ maintainers.cab404 ];
+
   options.services.pass-secret-service = {
     enable = mkEnableOption "Pass libsecret service";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.pass-secret-service;
+      defaultText = literalExpression "pkgs.pass-secret-service";
+      description = "Package providing <command>pass_secret_service</command>.";
+    };
+
+    storePath = mkOption {
+      type = types.str;
+      default = "$PASSWORD_STORE_DIR";
```

Comment:
Have you verified that this works? I suspect that it wouldn't work since the `PASSWORD_STORE_DIR` environment variable is not imported into the systemd session unless the X session is managed by HM. Even then it won't be imported until the graphical sessiont target is starting, i.e., long after the default target.

I think, instead, that it would make sense to omit the default value for the `storePath` variable. In the `password-store.nix` file add

    services.pass-secret-service.storePath = mkDefault cfg.settings.PASSWORD_STORE_DIR;

Then a user that has password-store enabled should get the correct path and other users will get an error unless they set a path themselves.

Reply:
Propagating the value in `password-store.nix` makes sense, though I wonder if it's good for `password-store.nix` to be setting its own default, since pass itself defaults to `~/.password-store`, and that's why we are forced to set the directory for pass_secret_service with `--path` regardless of the environment.

Either way, I'll definitely implement something else than what I have, like

```Nix
services.pass-secret-service.storePath = mkDefault cfg.settings.PASSWORD_STORE_DIR;
```

and have the `--path` arg only be added if `storePath` is not null or something.